proposals/stage-2/CPC_CHARTER/CPC-CHARTER.md

```diff
@@ -74,11 +74,11 @@ can be removed by voluntary resignation, or by a standard CPC motion.
 
 Voting members are selected as follows:
 
-* Each top level project may nominate two members through a process
+* Each Impact project may nominate two members through a process
```

Comment:
Feedback from JSF projects: We would like this to read "may nominate up to two"

Reply:
@jorydotcom added a commit to say up to two for all of the categories.